cmd/pr.go

```diff
@@ -13,20 +14,54 @@ import (
 var compareBranch string
 var title string
 var description string
+var number int
+var reviewers string
+var teamReviewers string
 
 func init() {
 	rootCmd.AddCommand(prCmd)
 	prCmd.Flags().StringVar(&compareBranch, "compare", "", "The branch to compare with base branch. Defaults to current local branch.")
 	prCmd.Flags().StringVar(&title, "title", "", "The title of the PR.")
+	prCmd.Flags().StringVar(&reviewers, "reviewers", "", "Add reviewers.")
+	prCmd.Flags().StringVar(&teamReviewers, "teamReviewers", "", "Add a team as reviewers.")
 	prCmd.Flags().StringVar(&description, "description", "", "The description of the PR.")
-	prCmd.MarkFlagRequired("title")
 }
 
 var prCmd = &cobra.Command{
 	Use:   "pr",
-	Short: "Create a pull request [github]",
-	Long:  `Create a pull request on github from compare branch to base branch`,
+	Short: "Create or show a pull request [github]",
+	Long: `Create a pull request on github from compare branch to base branch
+	ergo pr --title "my new pull request --reviewers pespantelis,nstratos,mantzas"
+Show details of a pr by pr number
+	ergo pr 18
+Add reviewers to an existing pr
+	ergo pr 18 --reviewers pespantelis,nstratos,mantzas"
+	`,
+	Args: cobra.MaximumNArgs(1),
 	RunE: func(cmd *cobra.Command, args []string) error {
+		var err error
```

Comment:
Not sure declaring error is necessary here. Assigning with `:=` and returning the error right away might be clearer (one less thing to remember).